[CONTENT] more wanderer jobs (#6083)
endless-sky/endless-sky: data/wanderer/wanderer jobs.txt

	on complete
		payment 35000 300
		dialog `When you return, the Wanderers thank you for taking them to the Kor Efreti and hand you <payment> before disembarking.`

mission "Captured soldiers"
	name "Transport captured Unfettered soldiers"
	job 
	repeat

> 	description "Transport <bunks> captured Unfettered soldiers to <planet>. payment is <payment>"

Reviewer: ```suggestion
	description "Transport <bunks> captured Unfettered soldiers to <planet>. Payment is <payment>."
```

	to offer
		has "language: Wanderer"
		random < 20
	passengers 3
	on offer
		require "Brig"
	source
		government "Wanderer"
	destination
		distance 2 8
		government "Wanderer"
	on complete
		payment
		payment 50000

> 		dialog "Once you land on <planet>, you watch as some Wanderers in uniform carry the Unfettered prisoners off of your ship. You collect your payment of <payment>."

Reviewer: ```suggestion
		dialog "Once you land on <planet>, several uniformed Wanderers drive the Unfettered prisoners off of your ship. You collect your payment of <payment>."
```

mission "Governor transport"
	name "transport the Governor to <planet>"
	job
	repeat

> 	description "transport the local governor to the capital city of Vara K'chrai. Payment is <payment>."

Reviewer: ```suggestion
	description "transport the local governor to the capital city of <planet>. Payment is <payment>."
```

(good practice to use the tokens anyway even if the destination is hardcoded)

	passengers 1
	deadline
	to offer
		has "language: Wanderer"

> 		random < 70

Reviewer: With such a high chance of offering this mission, people will wonder why Wanderer governors need to visit the capital so darn often. :) Consider reducing it a lot. If you want a high-chance-of-offering luxury accommodation mission, I would create a new one that's about someone lower in importance than a governor.
